pkg/virt-api/validating-webhook/validating-webhook_test.go

```diff
@@ -809,6 +810,47 @@ var _ = Describe("Validating Webhook", func() {
 			Expect(len(causes)).To(Equal(1))
 			Expect(causes[0].Field).To(Equal("fake[0].bootorder"))
 		})
+
+		It("should reject invalid SN characters", func() {
+			vmi := v1.NewMinimalVMI("testvmi")
+			order := uint(1)
+			sn := "$$$$"
+
+			vmi.Spec.Domain.Devices.Disks = append(vmi.Spec.Domain.Devices.Disks, v1.Disk{
+				Name:       "testdisk2",
+				VolumeName: "testvolume2",
+				BootOrder:  &order,
+				Serial:     sn,
+				DiskDevice: v1.DiskDevice{
+					Disk: &v1.DiskTarget{},
+				},
+			})
+
+			causes := validateDisks(k8sfield.NewPath("fake"), vmi.Spec.Domain.Devices.Disks)
+			Expect(len(causes)).To(Equal(1))
+			Expect(causes[0].Field).To(Equal("fake[0].serial"))
+		})
+
+		It("should reject SN > 36 characters", func() {
+			vmi := v1.NewMinimalVMI("testvmi")
+			order := uint(1)
+			sn := strings.Repeat("1", 257)
```

Comment:
this unit test is in the same package as the validating webhook code, so you can reference the maxStrLen var directly

```
sn := strings.Repeat("1", maxStrLen+1)
```

that way if the constant changes, the test keeps working :)